code/modules/antagonists/wizard/equipment/soulstone.dm

```diff
@@ -26,13 +26,17 @@
 /obj/item/soulstone/anybody
 	usability = TRUE
 
+/obj/item/soulstone/mystic
+	icon_state = "mystic_soulstone"
+	theme = "wizard"
```

Comment:
don't we generally lean towards using defines instead of strings for these kinds of things?
ie 
```
#define THEME_WIZARD "wizard"
theme = THEME_WIZARD
```

Reply:
Gonna take some time but I can do it.